poppler-sys/build.rs

```diff
@@ -110,13 +86,22 @@ fn builder() -> bindgen::Builder {
         .whitelist_function("poppler_.*")
         .whitelist_var("_?Poppler.*")
         .whitelist_var("_?poppler.*")
+        //
         .disable_name_namespacing()
+        .generate_comments(true)
+        .default_enum_style(bindgen::EnumVariation::Rust {
+            non_exhaustive: true,
+        })
+        //
+        .raw_line("use crate::dep_types::*;")
+        .raw_line("use crate::poppler::*;")
```

Comment:
Those imports are needed because of all blacklist that each module had when it got generated (because other types were already generated in other modules)

Since those imported types are used, they also need to be imported from the other rust modules